app/services/proofing/aamva/authentication_client.rb

```diff
@@ -2,28 +2,19 @@ module Proofing
   module Aamva
     class AuthenticationClient
       AAMVA_TOKEN_FRESHNESS_SECONDS = 28 * 60
+      AUTH_TOKEN_CACHE_KEY = 'aamva_api_auth_token'
 
-      class << self
-        attr_accessor :auth_token
-        attr_accessor :auth_token_expiration
-      end
-
-      def self.token_mutex
-        @token_mutex ||= Mutex.new
-      end
-
-      def fetch_token(config)
-        AuthenticationClient.token_mutex.synchronize do
-          if AuthenticationClient.auth_token.nil? || auth_token_expired?
-            send_auth_token_request(config)
-          end
-          AuthenticationClient.auth_token
+      def self.auth_token(config)
+        Rails.cache.fetch(
+          AUTH_TOKEN_CACHE_KEY, skip_nil: true,
+                                expires_in: AAMVA_TOKEN_FRESHNESS_SECONDS
```

Comment:
"all same line or all differnt lines"
```suggestion
          AUTH_TOKEN_CACHE_KEY,
          skip_nil: true,
          expires_in: AAMVA_TOKEN_FRESHNESS_SECONDS,
```